### DISCUSSION TITLE
Any way to plot a kde right now with the `seaborn.objects` interface?

### DISCUSSION THREAD

**Discussion by paw-lu** [mwaskom/seaborn#3103] [Q&A] [answered]
Learning the `seaborn.objects` interface. Is there any way to plot a kde right now using it?

Understand that it's new and a work in progress—just was curious if it was currently possible since how to accomplish it is not obvious.

---

Really enjoying learning the new interface!

**Comment by mwaskom** [comment, marked as answer]
Nope you haven't missed anything — I'd except a `KDE` object to appear in v0.12.2 though.

**Comment by mwaskom** [comment]
See #3111
